### PR TITLE
Update streamtypes overview

### PR DESCRIPTION
**Pull request by michaelsena**

### CODE REVIEW

**Comment by michaelsena** [comment]
@stbrody a simple change to add some more context to this page

File: docs/streamtypes/overview.md [hunk 1]
```diff
@@ -1,9 +1,24 @@
 # StreamTypes
+Each [stream](../learn/glossary.md#streams) on Ceramic must be created using a specific [StreamType](../learn/glossary.md#streamtypes). Different StreamTypes have different APIs for performing [writes](../build/writes.md). To create and/or update streams, see the API page for the respective StreamTypes below.
```
Comment (stbrody) [inline docs/streamtypes/overview.md:2]: reads and writes! Caip10Links can be queried using the `fromAccount` method

**Review by stbrody — APPROVED** [review]
lgtm mod note about streamtypes providing read apis also

File: docs/streamtypes/overview.md (hunk 1 above)
Comment (michaelsena) [reply docs/streamtypes/overview.md:2]: Is this true for all streamtypes or just CAIP10? I thought we wanted to keep the read APIs standardized so third-party developers don't need to care about what streamtype some other developer used when creating the data?

Comment (stbrody) [reply docs/streamtypes/overview.md:2]: `Ceramic.loadStream(<streamId>)` will work for any stream type.  But stream types may also expose additional special-purpose read APIs that are streamtype specific.  Deterministic document lookup within TileDocument can be thought of as another, and in fact we have plans to add a dedicated method for it: https://github.com/ceramicnetwork/js-ceramic/issues/1415

Comment (michaelsena) [reply docs/streamtypes/overview.md]: Thanks for the additional context. That's good to know

**Pull request merged**